python/seldon_core/microservice.py

```diff
@@ -316,21 +137,11 @@ def main():
 
     if args.persistence:
         logger.info('Restoring persisted component')
-        user_object = persistence.restore(user_class, parameters, debug=DEBUG)
+        user_object = persistence.restore(user_class, parameters)
```

Comment:
Please, consider refactoring `main()` as currently it's 136 lines and contians specific bits which could be separate methods.